src/Microsoft.AspNet.Diagnostics/ErrorPageMiddleware.cs

```diff
@@ -109,7 +115,7 @@ private Task DisplayException(HttpContext context, Exception ex)
                         Function = string.Empty
                     };
 
-                    ReadFrameContent(frame, fileContent, item.StartLine, item.EndLine);
+                    GetErrorCodeBlock(frame, fileContent, item.StartLine, item.EndLine);
```

Comment:
Good catch...I think I changed the existing behavior and have changed it in the latest iteration (although I am not sure how to reproduce a multi-line error...do you have any idea?)...I have tests covering these multiline scenarios, so I think we should be good.